docs/deployment/docker-compose.md

```diff
@@ -37,17 +38,18 @@ Run the user similator load test. For more information see [Load Test](#loadtest
 
 <!-- deploy-test-start run-tests -->
 
-    docker run weaveworksdemos/load-test -d 60 -h edge-router -c 3 -r 10
+    docker run --net mynetwork weaveworksdemos/load-test -d 60 -h edge-router -c 3 -r 10
```

Comment:
There's no --rm here, so the container won't be removed. Is that intended (just checking)?